CRM/Utils/System/WordPress.php

```diff
@@ -990,6 +995,13 @@ public function alterAssetUrl(\Civi\Core\Event\GenericHookEvent $e) {
     }
   }
 
+  /**
+   * @inheritDoc
+   */
+  public function checkPermissionAddUser() {
+    return current_user_can('create_users');
+  }
+
```

Comment:
This is all that's necessary for the "Create User Record" link to appear in the "Actions" menu.